styles/variables-silicon.less

```diff
@@ -345,9 +345,14 @@
 
 // Picker
 // ---------------------------------------
-@agate-picker-font-size: 27px;
+@agate-picker-font-size: 33px;
 @agate-picker-font-weight: normal;
```

Comment:
It seems that in the design the font-weight is bold.

Reply:
fixed